### DISCUSSION TITLE
Trojan detected on latest version 1.2.2

### DISCUSSION THREAD

**Discussion by hassansf** [rustdesk/rustdesk#5480] [General]
![image](https://github.com/rustdesk/rustdesk/assets/100449300/8db43746-95d6-4b35-9e74-922ab7091244)
Malware detected

![image](https://github.com/rustdesk/rustdesk/assets/100449300/342dab4e-169f-4660-8636-7c0349c03b3e)
No detection on previous version

Anyone care to explain? I am not taking any risk and make any exception in the AV.

**Comment by dinger1986** [comment]
https://github.com/rustdesk/rustdesk/discussions/5447

**Reply by dinger1986** [reply]
shall get an FAQ written up

**Reply by hassansf** [reply]
I've used several other remote access apps and none of them were detected as trojan. I am using my own relay server in my organization and there are over 30 devices with Rustdesk and Kaspersky installed. Should I make an exception in the AV server and wait for.........🙄

**Reply by dinger1986** [reply]
Others have been in the past, we are working on stopping bad actors using RustDesk

**Reply by hassansf** [reply]
I love Rustdesk, it made my life easier. I can't thank enough to the dev team.

**Reply by dinger1986** [reply]
Please report the false positives
